### PR TITLE
June newsletter

### PR DESCRIPTION
**Pull request by iskerrett**
June newsletter. Comments and feedback are welcome.

I still need to add the url for the release notes.

### CODE REVIEW

File: src/blog/2023/06/community-news-06.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Today we released <add link> FlowForge 1.8, featuring high availability for Node-RED and DevOps software delivery pipelines. Both these features were in high demand from our community and will make it easier to reliably deliver Node-RED for business critical applications.
+
+ Read about the details of FlowForge 1.8 in our release announcement. 
+
+## Upcoming events
+
+### Building Node-RED Applications for Scalability and High Availability
+
+Our June webinar will focus on the new FlowForge 1.8 of running high availability Node-RED applications. Marian Demme, FlowForge Product Manager, will lead this session and share practical insights and best practices to show how FlowForge can unlock the true potential of Node-RED in large-scale deployments.
```
Comment (robmarcer) [inline src/blog/2023/06/community-news-06.md:27]:
```suggestion
Our June webinar will focus on the new FlowForge 1.8 feature of running high availability Node-RED applications. Marian Demme, FlowForge Product Manager, will lead this session and share practical insights and best practices to show how FlowForge can unlock the true potential of Node-RED in large-scale deployments. 
```

File: src/blog/2023/06/community-news-06.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Gerrit Riessen has published a list of [Pros and Cons for using Node-RED](https://gorenje.medium.com/fourteen-for-fourteen-against-why-i-love-hate-and-connect-with-node-23797f9466ec). It is a pretty comprehensive list so check it out. FlowForge is working to address some of the cons in Gerrit's list, specifically software delivery pipelines and the ability to deploy out to many end points.
+
+
+
+## Join Our Team
+FlowForge is expanding our team. Check out the current openings:
+
+* [DevOps Engineer](https://boards.greenhouse.io/flowforge/jobs/4796271004)
+
```
Comment (robmarcer) [inline src/blog/2023/06/community-news-06.md:63]:
```suggestion
* [Sales Representative](https://boards.greenhouse.io/flowforge/jobs/4843566004)
```

File: src/blog/2023/06/community-news-06.md (hunk 1 above)
Comment (robmarcer) [reply src/blog/2023/06/community-news-06.md:27]: My brain is tired but I think my edit here is right?

File: src/blog/2023/06/community-news-06.md (hunk 2 above)
Comment (robmarcer) [reply src/blog/2023/06/community-news-06.md:63]: Should we not have both jobs?

**Review by robmarcer — CHANGES_REQUESTED** [review]

File: src/blog/2023/06/community-news-06.md (hunk 2 above)
Comment (iskerrett) [reply src/blog/2023/06/community-news-06.md:63]: added Sales Rep job.  It wasn't actually listed when I checked. Now I see it.

**Review by robmarcer — APPROVED** [review]

**Pull request merged**